pkg/api/core/v1/models/models.go

```diff
@@ -128,6 +128,12 @@ type ApplicationCreateRequest struct {
 	Configuration ApplicationUpdateRequest `json:"configuration" yaml:"configuration,omitempty"`
 }
 
+// ApplicationRestartRequest represents and contains the data needed to
```

Comment:
This looks to be not used. Which is sensible, because the app name info is provided as part of the request url.
IMHO this should be removed.